pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -841,6 +841,9 @@ type InstallConfig struct {
 	//     - false
 	//     - no
 	InstallWipe bool `yaml:"wipe"`
+	//   description: |
+	//     Indicates if MBR partition should be marked as bootable (active).
+	InstallMarkMBRBootable bool `yaml:"markMBRBootable,omitempty"`
```

Comment:
Since this is user facing, I don't know if this is the best name for it. I also don't have a better idea. I would need to look into the nomenclature/wording that is used within the industry. Maybe @Ulexus has thoughts?

Reply:
sure, I'm open for suggestions :)

Reply:
Yeah, and yaml tag can be `legacyBIOSSupport` as well.